code/modules/mob/living/simple_animal/elementals/elemental.dm

```diff
@@ -0,0 +1,186 @@
+	name = "fire elemental"
+	desc = "A raging inferno, encased within a single form."
+	speak_emote = list("fires")
+	icon_state = "fire"
+	icon_living = "fire"
+
+	melee_damage_upper = 15
+	melee_damage_lower = 15
+	attacktext = "scorches"
+	attack_sound = 'sound/magic/fireball.ogg'
+	damtype = BURN
+	playstyle_string = "<b>You are a fire elemental. You are a very well-rounded minion, and your abilities can be used to attack and defend in many situations.</b>"
+	deathmessage = "gutters, flickers, shrinks, and dies."
+
+/mob/living/simple_animal/elemental/water //All-rounder; utility
+	name = "water elemental"
+	desc = "A large form of water that shimmers in the light."
+	speak_emote = list("splashes")
+	icon_state = "water"
+	icon_living = "water"
+
+	harm_intent_damage = 0
+	attacktext = "splashes"
+	attack_sound = 'sound/magic/exit_blood.ogg'
+	playstyle_string = "<b>You are a water elemental. You are well-rounded, and your abilities can be used to influence many things around you.</b>"
+	deathmessage = "falls into a formless pile of water."
+	ventcrawler = 1
+
+/mob/living/simple_animal/elemental/life //Healing; defensive
+	name = "nature elemental"
+	desc = "A walking plant with swivelling stems."
+	speak_emote = list("creaks", "pines")
+	icon_state = "life"
+	icon_living = "life"
+
+	melee_damage_upper = 5 //Based around healing so he doesn't hit very hard
+	melee_damage_upper = 5
+	attacktext = "thwacks"
+	playstyle_string = "<b>You are a nature elemental. You don't do much damage, but you have many useful healing abilities to keep your master and subordinates in the fight.</b>"
+	deathmessage = "collapses, withers, and dies."
+
+/mob/living/simple_animal/elemental/arcane //All-rounder; both
+	name = "mana elemental"
+	desc = "A hovering mass of shimmering, translucent crystals."
+	speak_emote = list("swoons")
+	icon_state = "mana"
+	icon_living = "mana"
+
+	attacktext = "claws"
+	playstyle_string = "<b>You are a mana elemental. A raw manifestation of magic, you can harness the aura to great extent. Your power will wax and wane depending on your master's living status.</b>"
+	deathmessage = "glows, flares, and breaks apart."
+
+/mob/living/simple_animal/elemental/unbound //Random; random
+	name = "unbound elemental"
+	desc = "Oh. God. What is this horrible abomination of magic? This poor thing probably lives its tortured existence in eternal agony."
+	speak_emote = list("screams", "cries", "screeches", "whimpers")
+	icon_state = "unbound"
+	icon_living = "unbound"
+
+	attacktext = "slices"
+	playstyle_string = "<b>You are an unbound elemental. Living a life in eternal agony, you are an aberration of all the elements. All of your stats are randomized and your abilities are also randomized.</b>"
+	deathmessage = "screams horribly, broken and awful, before mercifully breaking apart."
+
+/mob/living/simple_animal/elemental/unbound/New()
+	..()
+	melee_damage_upper = rand(1,30)
+	melee_damage_lower = rand(1,30)
```

Comment:
As far as I can tell, when attacking a mob it picks between lower and upper and uses that damage value. If you have 1 upper but 30 lower you still have the potential to hit hard.